app/models/course_module.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 class CourseModule < ApplicationRecord
+  attr_accessor :progress
   include OrderHelper
+  include CourseYearHelper
```

Comment:
I would put the accessor after the `includes`.

Aaand `CourseYearHelper` is not something I expect inside `course_module`. Maybe rename it to `CourseLessonProgressHelper`?